src/main/java/uk/ac/cam/cl/dtg/segue/api/GlossaryFacade.java

```diff
@@ -0,0 +1,167 @@
+    public final Response getTerms(@QueryParam("start_index") final String startIndex,
+                                   @QueryParam("limit") final String limit) {
+
+        Map<Map.Entry<BooleanOperator, String>, List<String>> fieldsToMatch = Maps.newHashMap();
+        fieldsToMatch.put(immutableEntry(BooleanOperator.AND, TYPE_FIELDNAME), Collections.singletonList("glossaryTerm"));
+
+        ResultsWrapper<ContentDTO> c;
+        try {
+            Integer resultsLimit = null;
+            Integer startIndexOfResults = null;
+
+            if (null != limit) {
+                resultsLimit = Integer.parseInt(limit);
+            } else {
+                resultsLimit = DEFAULT_RESULTS_LIMIT;
+            }
+
+            if (null != startIndex) {
+                startIndexOfResults = Integer.parseInt(startIndex);
+            } else {
+                startIndexOfResults = 0;
+            }
+
+            c = this.contentManager.findByFieldNames(this.contentIndex, fieldsToMatch, startIndexOfResults, resultsLimit);
+            return Response.ok(c).build();
+        } catch (NumberFormatException e) {
+            return new SegueErrorResponse(Status.BAD_REQUEST,
+                    "Unable to convert one of the integer parameters provided into numbers. "
+                            + "Params provided were: limit" + DEFAULT_RESULTS_LIMIT + " and startIndex " + 0, e).toResponse();
+        } catch (ContentManagerException e) {
+            return new SegueErrorResponse(Status.INTERNAL_SERVER_ERROR,
+                    "Content acquisition error.", e).toResponse();
+        }
+    }
+
+    /**
+     * Gets the current version of the segue application.
+     *
+     * @param term_id - The ID of the term to retrieve.
+     *
+     * @return segue version as a string wrapped in a response.
+     */
+    @GET
+    @Path("terms/{term_id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Get the term with the given id.")
+    public final Response getTermById(@PathParam("term_id") final String term_id) {
+
+        if (null == term_id) {
+            return new SegueErrorResponse(Status.BAD_REQUEST, "Please specify a term_id.").toResponse();
+        }
+
+        Content c = null;
+        try {
+            c = this.contentManager.getContentDOById(this.contentIndex, term_id);
+            if (null == c) {
+                SegueErrorResponse error = new SegueErrorResponse(Status.NOT_FOUND, "No glossary term found with id: " + term_id);
+                log.debug(error.getErrorMessage());
+                return error.toResponse();
+            }
+        } catch (NumberFormatException e) {
+            return new SegueErrorResponse(Status.BAD_REQUEST,
+                    "Unable to convert one of the integer parameters provided into numbers. "
+                            + "Params provided were: limit" + DEFAULT_RESULTS_LIMIT + " and startIndex 0", e).toResponse();
```

Comment:
This might just be a checked exception from getContentDOById(...) but you're not converting any parameters to numbers so this should never happen.